docs/book/v6/tutorials/create-book-module.md

```diff
@@ -7,65 +7,103 @@ The below files structure is what we will have at the end of this tutorial and i
+    │       ├── Entity/
+    │       │   └── Book.php
+    │       ├── Handler/
+    │       │   ├── GetBookCollectionHandler.php
+    │       │   ├── GetBookHandler.php
+    │       │   └── PostBookHandler.php
+    │       ├── InputFilter/
+    │       │   └── CreateBookInputFilter.php
+    │       ├── Service/
+    │       │   ├── BookService.php
+    │       │   └── BookServiceInterface.php
+    │       ├── ConfigProvider.php
+    │       └── RoutesDelegator.php
+    ├── Core/
+    │   └── src/
+    │       └── Book/
+    │           └── src/
+    │               ├──Entity/
+    │               │   └──Book.php
+    │               ├──Repository/
+    │               │   └──BookRepository.php
+    │               └── ConfigProvider.php
+    └── App/
+        └──src/
+           └── InputFilter/
+               └── Input/
+                   ├── AuthorInput.php
+                   ├── NameInput.php
+                   └── ReleaseDateInput.php
 ```
 
 * `src/Book/src/Collection/BookCollection.php` - a collection refers to a container for a group of related objects, typically used to manage sets of related entities fetched from a database
-* `src/Book/src/Entity/Book.php` - an entity refers to a PHP class that represents a persistent object or data structure
-* `src/Book/src/Handler/BookHandler.php` - handlers are middleware that can handle requests based on an action
-* `src/Book/src/Repository/BookRepository.php` - a repository is a class responsible for querying and retrieving entities from the database
+* `src/Core/src/Book/src/Entity/Book.php` - an entity refers to a PHP class that represents a persistent object or data structure
+* `src/Book/src/Handler/GetBookCollectionHandler.php` - handler that reflects the GET action for the BookCollection class
+* `src/Book/src/Handler/GetBookHandler.php` - handler that reflects the GET action for the Book entity
+* `src/Book/src/Handler/PostBookHandler.php` - handler that reflects the POST action for the Book entity
+* `src/Core/src/Book/src/Repository/BookRepository.php` - a repository is a class responsible for querying and retrieving entities from the database
 * `src/Book/src/Service/BookService.php` - is a class or component responsible for performing a specific task or providing functionality to other parts of the application
 * `src/Book/src/ConfigProvider.php` -  is a class that provides configuration for various aspects of the framework or application
 * `src/Book/src/RoutesDelegator.php` - a routes delegator is a delegator factory responsible for configuring routing middleware based on routing configuration provided by the application
-* `src/Book/src/InputFilter/BookInputFilter.php` - input filters and validators
-* `src/Book/src/InputFilter/Input/*` - input filters and validator configurations
+* `src/Book/src/InputFilter/CreateBookInputFilter.php` - input filters and validators
+* `src/Core/src/App/src/InputFilter/Input/*` - input filters and validator configurations
```

Comment:
```suggestion
* `src/Book/src/InputFilter/Input/*` - input fields used by the input filters
```
Generic inputs are located under [src/App/src/InputFilter/Input/](https://github.com/dotkernel/api/tree/6.0/src/App/src/InputFilter/Input)
and module-specific inputs under [src/`<Module>`/src/InputFilter/Input/](https://github.com/dotkernel/api/tree/6.0/src/Admin/src/InputFilter/Input),
but never in the Core directory.